src/installation/guides/raspberry-pi.md

```diff
@@ -0,0 +1,201 @@
+# Raspberry Pi
+
+As of 2017-11-30, Void Linux provides prebuilt Raspberry Pi images and rootfs
+tarballs. Compatible images and tarballs exist for RPi 1, RPi 2 and RPi 3, in
+both glibc and musl variants.
+
+You can either:
+
+- write a prebuilt image directly to your SD card (e.g. `dd if=<void-rpi.img>
+   of=/dev/sdX bs=4M ; sync`), but that might give you a limited partition
+   layout that has to be expanded manually afterwards;
+
+or:
+
+- prepare the partition layout of your SD card manually and extract a tarball to
+   your SD card, as described by this guide.
+
+After downloading the chosen file, make sure to check that the sha256 checksum
+is correct. It is available in the `sha256sums.txt` file.
```

Comment:
Image verification is explained elsewhere.

Reply:
Removed.